R/execution.r

```diff
@@ -272,7 +272,7 @@ execute = function(request) {
             output_handler = oh,
             stop_on_error = 1L),
         interrupt = function(cond) interrupted <<- TRUE,
-        error = handle_error) # evaluate does not catch errors in parsing
+        error = .self$handle_error) # evaluate does not catch errors in parsing
```

Comment:
Before this change, `jupyter console --kernel ir32` (or ir3.3) worked, but `jupyter qtconsole --kernel  ir32` raised an error: https://github.com/IRkernel/IRkernel/issues/319#issuecomment-218404471